src/components/Table/GridTable.test.tsx

```diff
@@ -1023,6 +1024,42 @@ describe("GridTable", () => {
     // Then we still get back the header row
     expect(rows).toEqual([simpleHeader]);
   });
+
+  it("can apply emptyCell to missing content", async () => {
+    // Given the table with a specified emptyCell
+    const r = await render(
+      <GridTable<Row>
+        columns={[nameColumn, valueColumn]}
+        style={{ emptyCell: <>empty</> }}
+        rows={[
+          simpleHeader,
+          // And some content is null, undefined, and empty strings
+          { kind: "data", id: "1", name: "", value: null },
+          { kind: "data", id: "2", name: "a", value: undefined },
+          { kind: "data", id: "3", name: "c", value: 1 },
+        ]}
+      />,
+    );
+    // Then the cells with missing content have the `emptyCell` node applied.
+    expect(cell(r, 1, 0).textContent).toBe("empty");
+    expect(cell(r, 1, 1).textContent).toBe("empty");
+    expect(cell(r, 2, 1).textContent).toBe("empty");
```

Comment:
Maybe go ahead and add assertions for the 2,0 and 3,0 and 3,1 cells?

Both as a pedantic sanity check that we don't literally output "empty" for _everything_ :-) but also I think it'd help the readability to have "a", "c" and "1" in here to guide the mental mapping of test input --> output assertions.